### ISSUE TITLE
Don't exist task haml:erb2haml

### ISSUE THREAD

**Issue by lancedikson** [indirect/haml-rails#86]
So, I don't have haml:erb2haml task in rake task list.

```
➜ ~/Projects/awedd (master) rake haml:erb2haml
rake aborted!
Don't know how to build task 'haml:erb2haml'

(See full trace by running task with --trace)
```

Gem was installed with bundler: I wrote ```gem "haml-rails"``` to Gemfile and ran ```bundle```.

```
Rails 4.2.0
ruby 2.2.0p0 (2014-12-25 revision 49005) [x86_64-darwin14]

Gems included by the bundle:
  * actionmailer (4.2.0)
  * actionpack (4.2.0)
  * actionview (4.2.0)
  * activejob (4.2.0)
  * activemodel (4.2.0)
  * activerecord (4.2.0)
  * activesupport (4.2.0)
  * arel (6.0.0)
  * bcrypt (3.1.9)
  * binding_of_caller (0.7.2)
  * builder (3.2.2)
  * bundler (1.7.12)
  * byebug (3.5.1)
  * capistrano (3.3.5)
  * capistrano-bower (1.0.0)
  * capistrano-bundler (1.1.4)
  * capistrano-rails (1.1.2)
  * capistrano-stats (1.1.1)
  * colorize (0.7.5)
  * columnize (0.9.0)
  * debug_inspector (0.0.2)
  * debugger-linecache (1.2.0)
  * docile (1.1.5)
  * erubis (2.7.0)
  * execjs (2.2.2)
  * fabrication (2.12.2)
  * faker (1.4.3)
  * globalid (0.3.0)
  * haml (4.0.6)
  * haml-rails (0.8.0)
  * hike (1.2.3)
  * html2haml (2.0.0)
  * i18n (0.7.0)
  * jbuilder (2.2.6)
  * json (1.8.2)
  * kgio (2.9.3)
  * loofah (2.0.1)
  * mail (2.6.3)
  * mime-types (2.4.3)
  * mini_portile (0.6.2)
  * minitest (5.5.1)
  * multi_json (1.10.1)
  * net-scp (1.2.1)
  * net-ssh (2.9.2)
  * nokogiri (1.6.5)
  * rack (1.6.0)
  * rack-test (0.6.3)
  * rails (4.2.0)
  * rails-deprecated_sanitizer (1.0.3)
  * rails-dom-testing (1.0.5)
  * rails-html-sanitizer (1.0.1)
  * railties (4.2.0)
  * raindrops (0.13.0)
  * rake (10.4.2)
  * rdoc (4.2.0)
  * ruby_parser (3.6.4)
  * sass (3.4.10)
  * sass-rails (5.0.1)
  * sdoc (0.4.1)
  * sexp_processor (4.4.5)
  * simplecov (0.9.1)
  * simplecov-html (0.8.0)
  * slop (3.6.0)
  * spring (1.2.0)
  * sprockets (2.12.3)
  * sprockets-rails (2.2.2)
  * sqlite3 (1.3.10)
  * sshkit (1.6.1)
  * thor (0.19.1)
  * thread_safe (0.3.4)
  * tilt (1.4.1)
  * tzinfo (1.2.2)
  * uglifier (2.7.0)
  * unicorn (4.8.3)
  * web-console (2.0.0)
```

**Comment by indirect** [comment]
This should be fixed in 0.8.2, which I just released. Thanks for reporting!

**Comment by lancedikson** [comment]
@indirect yeah, now it's fine. Thanks.
